fails:Hash#initialize_copy raises a TypeError if called on a frozen instance
frozen:Hash#initialize_copy raises a TypeError if called on a frozen instance
fails:Hash#initialize_copy is private
fails:Hash#initialize_copy raises a RuntimeError if called on a frozen instance
